src/views/AllocRoundListView.js

```diff
@@ -3,7 +3,6 @@ import { useEffect, useState } from "react";
 import { useNavigate } from "react-router-dom";
 import dao from "../ajax/dao";
 import Logger from "../logger/logger";
-
```

Comment:
These empty lines between import statements were added (in #236) for a reason: to group non-component and component imports to the separate groups. There’s much more component import than the other, so this grouping makes it easier to see what’s imported.

Reply:
However the other changed are good, so I’m merging this and re-adding the grouping lines back in the first commit of #238.